docs/content/environments.md

```diff
@@ -257,7 +253,6 @@ type = "webpack"
 account_id = "12345678901234567890"
 zone_id = "09876543210987654321"
 route = "example.com/*"
-workers_dev = false
```

Comment:
Currently we only require `workers_dev` in the top level due to the ambiguous nature of the `--release` flag. Since we are deprecating `--release`, this is no longer necessary since we know that if there is a `route` present, they are not deploying to their workers.dev subdomain

Reply:
Agreed we should make this clear in the changelog